src/LazyOperations/LinearMap.jl

```diff
@@ -362,24 +362,15 @@ A list of vertices.
 
 ### Algorithm
 
-We assume that the underlying set `X` is polyhedral.
-Then the result is just the linear map applied to the vertices of `X`.
+We assume that the underlying set `X` is polyhedral, and compute the list of vertices
+of `X`. The result is just the linear map applied to each vertex of `X`.
 """
-function vertices_list(lm::LinearMap{N}; prune::Bool=true)::Vector{Vector{N}} where {N<:Real}
-    # for a zero map, the result is just the list containing the origin
-    if iszero(lm.M)
-        return [zeros(N, dim(lm))]
-    end
-
-    # collect low-dimensional vertices lists
+function vertices_list(lm::LinearMap; prune::Bool=true)
+    # collect list of vertices of the wrapped set
```

Comment:
i remove the branch that checks if the user wants to compute `M * X` for `M` the identically null matrix. this case is handled by the prune.

Reply:
... and also because it introduces a type instability whenever the type of the vertices is not the same as `Vector{N}`.